static/js/publisher/market/stickyListingBar.js

```diff
@@ -0,0 +1,10 @@
+export default function() {
+  document.addEventListener("scroll", function() {
```

Comment:
Please use `debounce` to limit number of calls of `scroll` event.

See example of it in: https://github.com/canonical-websites/snapcraft.io/blob/master/static/js/public/fsf-language-select.js#L59-L60